From c618d1b2ff7451cc800ae08bcec67918f2eb5ba4 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Mon, 22 Jan 2024 15:54:03 +0100
Subject: [PATCH 0581/1695] dts: stm32: vdd_sdcard not always-on for
 stm32mp257f-ev1

Remove always-on on vdd_sdcard power supplies in order
to reduce power consumption.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git c67935a620eb65cc08f05c4f2f6632043bd1857a dts: stm32: vdd_sdcard not always-on for stm32mp257f-ev1]
Change-Id: I408596faa67e7e03007539636773f5afbca4aac7
Signed-off-by: Pascal Paillet <p.paillet@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/353320
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 core/arch/arm/dts/stm32mp257f-ev1.dts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp257f-ev1.dts b/core/arch/arm/dts/stm32mp257f-ev1.dts
index dbeb892be..6cb066677 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp257f-ev1.dts
@@ -287,7 +287,6 @@
 				regulator-name = "vdd_sdcard";
 				regulator-min-microvolt = <3300000>;
 				regulator-max-microvolt = <3300000>;
-				regulator-always-on;
 				regulator-over-current-protection;
 				st,regulator-bypass-microvolt = <3300000>;
 				st,pwrctrl_sel = <3>;
-- 
2.47.2

